ugvc/mrd/balanced_strand_utils.py

```diff
@@ -1224,7 +1682,10 @@ def convert_h5_to_papyrus_json(h5_file: str, output_json: str) -> str:
         h5_file_keys = store.keys()
```

Comment:
Maybe the last three functions belong to metrics_utils? We already have a function to convert h5_to_papyrus_json, no? Can't be used?

Reply:
Maybe it would be better to work with this function, I think I tried this and had some issues. This is not a new piece of code, so I am deferring this to a later improvements in the statistics this produces.